apps/backend/src/data/migrations/1654699651443-create-store-table.ts

```diff
@@ -0,0 +1,42 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class CreateStoreTable1654699651443 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'store',
+        columns: [
+          {
```

Comment:
`isNullable: true` means that this field can be null, which is not the case for this field. The only `isNullable: true` field is description in this table

Reply:
Done. We understood oposite. We thought that when we set isNullable, it means that we activate this option in order to not allow null value